kafka/consumer/group.py

```diff
@@ -267,6 +269,18 @@ def __init__(self, *topics, **configs):
                         new_config, self.config['auto_offset_reset'])
             self.config['auto_offset_reset'] = new_config
 
+        request_timeout_ms = self.config['request_timeout_ms']
+        session_timeout_ms = self.config['session_timeout_ms']
+        fetch_max_wait_ms = self.config['fetch_max_wait_ms']
+        if request_timeout_ms <= session_timeout_ms:
+            raise KafkaConfigurationError(
+                "Request timeout (" + str(request_timeout_ms) + ") must be larger than session timeout (" +
```

Comment:
`"Request timeout (%s) must be larger than session timeout (%s)" % (request_timeout_ms, session_timeout_ms)` is another alternative here